ckan/lib/helpers.py

```diff
@@ -813,8 +817,12 @@ def _make_menu_item(menu_item, title, **kw):
 
     This function is called by wrapper functions.
     '''
+    # display the menu item if it comes from flask
+    if '.' in menu_item:
+        link = _link_to(title, menu_item, suppress_active_class=True)
+        return literal('<li>') + link + literal('</li>')
```

Comment:
yes i agree about this, i've added just for testing to keep the flow going with less modifications possible